routes/modelExport.js

```diff
@@ -1,9 +1,33 @@
-var linz = require('../');
+'use strict';
+
+const linz = require('../');
+const setTemplateScripts = require('../lib/scripts');
```

Comment:
My vision for Linz is to become one big API. All of the pages that we deliver should use that API. Down the track, I want to allow someone to override the HTML for any page within Linz. Having the ability to spit out dependent CSS and JS in the same fashion as the rest of Linz works would be important. Does that make sense?

I don't think we need to expose `res.locals` via the API, because programmers have access to Express and can set it themselves?